src/components/Swap/Toolbar/Caption.tsx

```diff
@@ -62,9 +60,7 @@ function Caption({ icon: Icon = AlertTriangle, caption, color = 'secondary', too
 
 function GasEstimate({ gasUseEstimateUSD }: GasEstimateProps) {
   const isWideWidget = useIsWideWidget()
-  const displayEstimate = !gasUseEstimateUSD
-    ? '-'
-    : formatCurrencyAmount({ amount: gasUseEstimateUSD, isUsdPrice: true })
+  const displayEstimate = formatCurrencyAmount(gasUseEstimateUSD, NumberType.FiatGasPrice)
```

Comment:
This feels like a dangerous change to make in the same PR. Are you able to move it to its own PR, once conedison lands?
What does formatCurrencyAmount return if !gasUseEstimateUSD?